build.sh

```diff
@@ -1,2 +1,2 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
`env` has messed me up quite a few times and it's required to be installed
I think keeping it as `#!/bin/bash` is better

Reply:
Switched it to use `/bin/sh` as the script is POSIX-compliant, and that is the most portable solution that angers no one :wink: